--- aws-xray-sdk-2.6.0/setup.py.orig	2020-06-05 16:07:23.000000000 +0200
+++ aws-xray-sdk-2.6.0/setup.py	2021-02-16 18:39:10.099640176 +0100
@@ -47,7 +47,7 @@
         'jsonpickle',
         'enum34;python_version<"3.4"',
         'wrapt',
-        'future',
+        'future;python_version<"3.4"',
         'botocore>=1.11.3',
     ],
 
--- aws-xray-sdk-2.6.0/aws_xray_sdk/core/plugins/ec2_plugin.py.orig	2020-06-05 16:07:23.000000000 +0200
+++ aws-xray-sdk-2.6.0/aws_xray_sdk/core/plugins/ec2_plugin.py	2021-02-16 18:39:50.929418982 +0100
@@ -1,9 +1,11 @@
 import json
 import logging
-from future.standard_library import install_aliases
+import sys
 from urllib.request import urlopen, Request
 
-install_aliases()
+if sys.version_info < (3, 4):
+    from future.standard_library import install_aliases
+    install_aliases()
 
 log = logging.getLogger(__name__)
 
--- aws-xray-sdk-2.6.0/aws_xray_sdk/ext/sqlalchemy/util/decorators.py.orig	2020-06-05 16:07:23.000000000 +0200
+++ aws-xray-sdk-2.6.0/aws_xray_sdk/ext/sqlalchemy/util/decorators.py	2021-02-16 18:40:42.962470428 +0100
@@ -1,8 +1,12 @@
 import re
+import sys
 from aws_xray_sdk.core import xray_recorder
 from aws_xray_sdk.ext.util import strip_url
-from future.standard_library import install_aliases
-install_aliases()
+
+if sys.version_info < (3, 4):
+    from future.standard_library import install_aliases
+    install_aliases()
+
 from urllib.parse import urlparse, uses_netloc
 from sqlalchemy.engine.base import Connection
 
